api/cmd/transformer/.env.sample

```diff
@@ -2,10 +2,12 @@ LOG_LEVEL=ERROR
 
 STANDARD_TRANSFORMER_CONFIG={"transformerConfig":{"preprocess":{"inputs":[{"variables":[{"name":"customer_id","jsonPath":"$$.customer.id"}]},{"tables":[{"name":"driver_table","baseTable":{"fromJson":{"jsonPath":"$$.drivers[*]","addRowNumber":true}}}]},{"feast":[{"project":"merlin","entities":[{"name":"merlin_test_driver_id","valueType":"STRING","jsonPath":"$$.drivers[*].id"}],"features":[{"name":"merlin_test_redis_driver_features:completion_rate","valueType":"DOUBLE","defaultValue":"0.0"},{"name":"merlin_test_redis_driver_features:cancellation_rate","valueType":"DOUBLE","defaultValue":"0.0"}],"tableName":"driver_redis_feature_table","source":"REDIS"},{"project":"merlin","entities":[{"name":"merlin_test_driver_id","valueType":"STRING","jsonPath":"$$.drivers[*].id"}],"features":[{"name":"merlin_test_bt_driver_features:rating","valueType":"DOUBLE","defaultValue":"0.0"}],"tableName":"driver_bigtable_feature_table","source":"BIGTABLE"}]}],"transformations":[{"tableTransformation":{"inputTable":"driver_table","outputTable":"driver_table","steps":[{"sort":[{"column":"row_number","order":"DESC"}]},{"renameColumns":{"id":"merlin_test_driver_id","row_number":"rank"}},{"updateColumns":[{"column":"customer_id","expression":"customer_id"}]},{"selectColumns":["customer_id","merlin_test_driver_id","rank"]}]}},{"tableJoin":{"leftTable":"driver_table","rightTable":"driver_redis_feature_table","outputTable":"result_table","how":"LEFT","onColumns":["merlin_test_driver_id"]}},{"tableJoin":{"leftTable":"result_table","rightTable":"driver_bigtable_feature_table","outputTable":"result_table","how":"LEFT","onColumns":["merlin_test_driver_id"]}},{"tableTransformation":{"inputTable":"result_table","outputTable":"result_table","steps":[{"sort":[{"column":"rank"}]},{"renameColumns":{"merlin_test_driver_id":"driver_id"}},{"selectColumns":["rank","driver_id","customer_id","merlin_test_redis_driver_features:completion_rate","merlin_test_redis_driver_features:cancellation_rate","merlin_test_bt_driver_features:rating"]}]}}],"outputs":[{"jsonOutput":{"jsonTemplate":{"fields":[{"fieldName":"instances","fromTable":{"tableName":"result_table","format":"SPLIT"}}]}}}]},"postprocess":{}}}
 
-MERLIN_TRANSFORMER_MODEL_NAME=echo
-MERLIN_TRANSFORMER_MODEL_PREDICT_URL=localhost:8082
-MERLIN_TRANSFORMER_PORT=8081
+CARAML_MODEL_FULL_NAME=echo
+HTTP_PORT=8081
```

Comment:
Why is it `HTTP_PORT` and `GRPC_PORT` instead of `CARAML_HTTP_PORT` and `CARAML_GRPC_PORT` ?

Reply:
no reason, I paste the wrong env name